packages/eas-cli/src/fetch.ts

```diff
@@ -8,6 +10,16 @@ export class RequestError extends Error {
   }
 }
 
+function createHttpAgent(): Agent | null {
+  const httpProxyUrl = process.env.http_proxy;
```

Comment:
Is it some kind of standard that this is in lowercase (`http_proxy`)?

Reply:
yes, but upper case also is a valid option, in our docs we are listing multiple value that should be set. npm rc is using lowercase value so i decided to go with that here

https://docs.expo.dev/guides/troubleshooting-proxies/#for-command-line-applications